README.md

```diff
@@ -1,6 +1,8 @@
 Zinc
 ====
 
+**This is Scala Center's fork of [Zinc][sbt/zinc].**
```

Comment:
Sorry, this change was not supposed to be added to this pull request. I'm removing this and I'll add it afterwards. I'm moving my private repository to scala center's organization before public release of blog post.